eve/workers/end2end/requirements.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+KUBECTL_VERSION=1.18.3
+KIND_VERSION=v0.11.1
+HELM_VERSION=v3.2.3
```

Comment:
this version might be a little old? I have 3.7 locally.

Reply:
I'm using the same helm version that was defined before https://github.com/scality/Zenko/pull/1273/files#diff-9c090f3622e1659481c1d1c358eb699844a94ad8c711d352f730f34f9004b061L7

I'm trying to not make too many modification to the environment. However I did modify the version of kind because the one we used before didn't worked with the new env.